### ISSUE TITLE
Peak center time out of peak bounds.

### ISSUE THREAD

**Issue by WenzDaniel** [XENONnT/straxen#1284]
**Describe the bug**
As observed by @jjakob03 there is a bug in straxen which leads to peak center times out of peak bounds. See also the [chat](https://xenonnt.slack.com/archives/C016DM0JPK9/p1698327543070729).

**Comment by JYangQi00** [comment]
I have plotted the raw records per channel for this S1 peak from the conversation on Slack and I see something interesting. It seems to me that the start of the raw records does not start from a value near baseline. I checked to see if there were any raw records before this problematic S1 peak, but it doesn't really seem so (see plots attached, the second plot is just a zoomed out version of the first with some other records in there).
![image](https://github.com/XENONnT/straxen/assets/40413412/6016f0ac-3cad-4a0e-9700-0c41171c666c)
![image](https://github.com/XENONnT/straxen/assets/40413412/ef8581cc-3da8-4907-89e4-c140e8df34a7)

Given that these raw records don't have a baseline, then such a baseline is calculated when processing from raw records to records. However, the baseline calculation is just the average of the first 40 samples. If the first 40 samples start far from the true baseline, then the inversion of the records will have a huge negative component. This event is removed by the DAQ veto cut anyway, so I will close this issue.
